server.js

```diff
@@ -78,15 +78,22 @@ app.get('/users/add', (req, res) => {
   });
 });
 
-/* fake data for backend work */
-app.get('/api/events', (req, res) => {
-  const events = [
-    { id: 1, firstName: 'John', lastName: 'Doe' },
-    { id: 2, firstName: 'Joe', lastName: 'Swanson' },
-    { id: 3, firstName: 'Kanye', lastName: 'West' },
-  ];
-
-  res.json(events);
+app.post('/postEvent', (req, res) => {
+  // POST request for new Event.
+  const {
+    // should be camelcase for style guide but needs to match table columns
+    team_name, team_id, date, time, location, description,
```

Comment:
what is the team_id? other than that everything looks good.

Reply:
team_id is just a numerical id that we have in the events table because it is part of the team primary key. We should probably make the primary key team_name, gender and drop id.